Dockerfile

```diff
@@ -0,0 +1,42 @@
+# this is mostly taken from the spm docker files: https://github.com/spm/spm-docker
```

Comment:
This is the recipe to follow to create the docker image. Let me know if it is unclear but I tired to add a few comments here and there.